app/views/components/inputRadio.scala.html

```diff
@@ -24,7 +24,6 @@
   fieldName: String,
```

Comment:
Why is this stuff here?

Reply:
not really sure tbh.  Some of my rebase attempts failed as had merge conflicts and so seems like it pushed up some of the changes from other PRs and marked them as done in this PR